src/Common/src/Abstractions/Util/SnakeCaseAllCapsEnumMemberTypeConverter.cs

```diff
@@ -36,11 +36,6 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
     /// <inheritdoc />
     public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
     {
```

Comment:
Is this because of nullable enable, I am not seeing where it is. Otherwise it seems like parse will fail, although we are not checking for other bad input.